src/widgets/_notebooks.scss

```
@@ -68,7 +68,7 @@ tab {
     min-width: 24px;
     padding: rem(3px);
 
-    .tab-close-button {
+    .tab-close-button, .tab-indicator{
```

Thanks, this was simple. :sweat_smile: One style nit: we put selectors on their own lines, and leave spaces between selectors and the opening brace.

```suggestion
    .tab-close-button,
    .tab-indicator {
```

---

Yeah, that way looks clearer. Thanks for the tip!